### PR TITLE
code cleanup for container/helpers.go

### PR DESCRIPTION
**Pull request by victory460**
#### What type of PR is this?

/kind cleanup

 
 #### Does this PR introduce a user-facing change?

```release-note
 NONE
```

### CODE REVIEW

File: pkg/kubelet/container/helpers.go [hunk 1]
```diff
@@ -277,14 +277,6 @@ func SandboxToContainerState(state runtimeapi.PodSandboxState) State {
 	return ContainerStateUnknown
 }
 
-// FormatPod returns a string representing a pod in a human readable format,
-// with pod UID as part of the string.
-func FormatPod(pod *Pod) string {
```
Comment (ehashman) [inline pkg/kubelet/container/helpers.go:282]: Code search confirms this isn't used, wonder why the linter didn't catch this.

https://cs.k8s.io/?q=FormatPod&i=nope&files=&excludeFiles=&repos=kubernetes/kubernetes

**Review by ehashman — COMMENTED** [review]
/priority backlog
/lgtm
/triage accepted

File: pkg/kubelet/container/helpers.go (hunk 1 above)
Comment (victory460) [reply pkg/kubelet/container/helpers.go:282]: Yes `FormatPod` isn't used. We checked `.golangci.yaml` but did not find the cause of the undetected problem . We will continue to follow up on this linter problem when have time. Thanks a lot for your review.

**Comment by dims** [comment]
@klueska @mrunalp : this needs a approval, just removes some dead code etc. very quick review!

**Pull request merged**